esphome/core_config.py

```diff
@@ -160,7 +160,7 @@ def valid_include(value):
         cv.Optional(CONF_ON_BOOT): automation.validate_automation(
             {
                 cv.GenerateID(CONF_TRIGGER_ID): cv.declare_id(StartupTrigger),
-                cv.Optional(CONF_PRIORITY, default=600.0): cv.float_,
+                cv.Optional(CONF_PRIORITY, default=399.0): cv.float_,
```

Comment:
Reverting this so we can put the update fix in
```suggestion
                cv.Optional(CONF_PRIORITY, default=600.0): cv.float_,
```